clients/cloud/python/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Produce messages to and consume messages from Confluent Cloud using Confluent Python Client for Apache Kafka
+
+
+# Prerequisites
+
+* Install Confluent's Python Client for Apache Kafka: https://github.com/confluentinc/confluent-kafka-python
+* File with client configuration parameters to connect to your Confluent Cloud instance
+
+
+# Usage
+
+$ cat ~/.ccloud/librdkafka.config
+bootstrap.servers=<>
+sasl.username=<>
+sasl.password=<>
+
+
+$ ./producer.py ~/.ccloud/librdkafka.config test1                      
+alice 	 {"count": 0}
+alice 	 {"count": 1}
+alice 	 {"count": 2}
+alice 	 {"count": 3}
+alice 	 {"count": 4}
+alice 	 {"count": 5}
+alice 	 {"count": 6}
+alice 	 {"count": 7}
+alice 	 {"count": 8}
+alice 	 {"count": 9}
+
+
+$ ./consumer.py ~/.ccloud/librdkafka.config test1                       
```

Comment:
And... ```
gwenshap-mac:python gwen$ ./consumer.py ~/.ccloud/librdkafka.config test1
end of partition: test1 [0] @ 10

```

I think I really failed to connect to my cloud instance and need to check, but not that the producer script gave me both errors and success indicators, so it was difficult to tell.

Reply:
`auto.offset.reset` defaults to latest. We could use `assign` or just default to `earliest`. Alternatively the demo could start the consumer first, place it in the background. 

`./consumer.py ~/.ccloud/librdkafka.config test1 &`

Reply:
Sorry for dupe comment, but adding here in the thread (I couldn't comment here before...it was strange)

The consumer code already had set 

```bash
'auto.offset.reset': 'earliest'
```

https://github.com/confluentinc/examples/blob/DEVX-543/clients/cloud/python/consumer.py#L56

This is at odds with the behavior @gwenshap encountered, because the consumer should have started at the beginning of the log.

@edenhill , @rnpridgeon : is there some difference in behavior between versions that would explain this?

Reply:
Note that the issue here was fixed by adding:        
```
 'default.topic.config': {'auto.offset.reset': 'smallest'}
```
to my configuration

Reply:
My issue occurred with confluent-kafka 0.11.5. It was resolved (without the extra config) when I upgraded to 0.11.6.

I suggest documenting that this example works with confluent-kafka 0.11.6.

Reply:
Recommendation is going to be 1.0